basic/settings.py

```diff
@@ -14,9 +14,7 @@
     }
 }
 
-ALLOWED_HOSTS = [
-    "localhost",
-]
+ALLOWED_HOSTS = ['*']
```

Comment:
Setting this to a wildcard disables [security protection on the Host header](https://docs.djangoproject.com/en/2.1/ref/settings/#allowed-hosts). 

It would probably be better to make this an environment variable that supplies the list of allowed hosts, defaulting to localhost when none is provided:

```python
ALLOWED_HOSTS = os.environ.get('DJANGO_ALLOWED_HOSTS', 'localhost').split(' ')
```

I'm not sure what that would mean on the ECS side of things as far as setting that environment variable.